graph/schema.graphqls

```diff
@@ -3,28 +3,39 @@ Copyright Contributors to the Open Cluster Management project.
 
 Search Query API.
 """
-schema { 
+schema {
   query: Query
+  subscription: Subscription
+}
+
+"""
+`Subscriptions` supported by the Search Query API.
+"""
+type Subscription {
+  """
+  searchSubscription will return a stream of `SearchResult` objects.
+  """
+  searchSubscription(input: [SearchInput]): [SearchResult]
```

Comment:
The name `searchSubscription` sounds redundant.  Can we name it just `search` or will it create a conflict with the Query with the same name?

Reply:
Need to test a little more, but this seems to be fine as search is contained within the Subscription type.